### PR TITLE
feat(v9): migrate checkbox,input,label,link,select to ship rolluped only dts

### PR DESCRIPTION
**Pull request by Hotell**
Applied `yarn workspace-generator migrate-converged-pkg` to ship only rolluped type definitions for:
- react-checkbox
- react-input
- react-label
- react-link
- react-select

## Related Issue(s)

Fixes partially https://github.com/microsoft/fluentui/issues/22429

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                       |    Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                  Change |
| :-------------------------------------------------------------------------------------- | --------------------------: | --------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr> | `23.556 kB`<br />`7.898 kB` | `23.534 kB`<br />`7.892 kB` | `-22 B` <br />`-6 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                 |        Size (minified/GZIP) |
| ------------------------------------------------------------------------------------------------- | --------------------------: |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr> |  `8.205 kB`<br />`3.516 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>        | `24.401 kB`<br />`8.024 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/837c5ffda3334356cc26dab37e9b095edc5d3da0'>837c5ffda3334356cc26dab37e9b095edc5d3da0</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/Avatar-mount_base.html">1221</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/Avatar-mount.html">1172</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/Button-mount_base.html">734</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/Button-mount.html">774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProvider-mount_base.html">2347</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProvider-mount.html">2278</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProviderWithTheme-mount_base.html">382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProviderWithTheme-mount.html">372</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">313</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">445</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">409</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/MakeStyles-mount_base.html">2005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22793/perf-test-react-components/MakeStyles-mount.html">1924</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-input/.storybook/preview.js [hunk 1]
```diff
@@ -4,11 +4,4 @@ import * as rootPreview from '../../../../.storybook/preview';
 export const decorators = [...rootPreview.decorators];
 
 /** @type {typeof rootPreview.parameters} */
-export const parameters = {
```
Comment (khmakoto) [inline packages/react-components/react-input/.storybook/preview.js:7]: This is changing only in `react-input`, is it intentional?

Comment (Hotell) [reply packages/react-components/react-input/.storybook/preview.js:7]: I have no context why there is used such a custom setup. Nothing was changed by hand from my side. If that's configuration has a reasoning I'd suggest to provide that reasoning for future migration runs, so person executing those automation has full context and thus can undo appropriately. migration generators are not using AST for storybook configs as that's not needed at the moment and to implement it the effort would be non trivial.

ty

**Review by khmakoto — APPROVED** [review]

**Review by TristanWatanabe — APPROVED** [review]

**Pull request merged**